### DISCUSSION TITLE
ERROR   -  Config value 'docs_dir': The path 'mkdocs/Docs' isn't an existing directory.

### DISCUSSION THREAD

**Discussion by Skb08** [mkdocs/mkdocs#3732] [Q&A]
Run mkdocs build -f mkdocs/mkdocs.yml
  mkdocs build -f mkdocs/mkdocs.yml
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.1[2](https://github.com/PranavBarthwal/cosmoXplore/actions/runs/9470405451/job/26091226787?pr=394#step:6:2).3/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/[3](https://github.com/PranavBarthwal/cosmoXplore/actions/runs/9470405451/job/26091226787?pr=394#step:6:3).12.3/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.3/x6[4](https://github.com/PranavBarthwal/cosmoXplore/actions/runs/9470405451/job/26091226787?pr=394#step:6:4)
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.3/x[6](https://github.com/PranavBarthwal/cosmoXplore/actions/runs/9470405451/job/26091226787?pr=394#step:6:6)4
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.3/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.3/x64/lib
ERROR   -  Config value 'docs_dir': The path 'mkdocs/Docs' isn't an existing directory.

Aborted with a configuration error!
Error: Process completed with exit code 1.

**Comment by pawamoy** [comment]
Directory names are case-sensitive. Specify `docs_dir: docs`, or rename your directory to `Docs`.

Also the path is relative to mkdocs.yml. Make sure to specify `docs_dir` accordingly.
